mern/client/src/utils/test_transcript.json

```diff
@@ -2029,6 +2029,3351 @@
         }
       ],
       "question_included": false
+    },
+    "q2": {
+      "question": "What kind of recipe are you looking for?",
+      "intents": [
+        {
```

Comment:
Inside each list exists JSON objects, however, we don't want to have an explicit title for each object (i.e. remove q1, q2 ... and intent1, intent2...). This makes list accessing by index possible and works with the parse transcript function.

For reference on how to change the structure, see the test_transcript file in the parseTranscript branch for reference.

Just as an example: we would want the current template of:
intents [
{
     intent1: { ....}
     intent2: {...}
}
]

to be changed into:

intents [
{ ... info for intent1},
{... info for intent2}
]